src/redux/reducers/index.js

```diff
@@ -4,13 +4,15 @@ import signUp from "./signupReducer";
 import languageReducer from "./languageReducer";
 import profileReducer from "./profileReducer";
 import roleReducer from "./roleReducer";
+import dashboardReducer from '../reducers/dashboardReducer';
```

Comment:
may you not class it dashboard for it's all about accommodations.

Reply:
@blaise82  Thanks for such good insight. I'll tweak it like so

Reply:
@blaise82  Thanks, feedback implemented